internal/services/recoveryservices/site_recovery_replication_policy_hyperv_resource.go

```diff
@@ -0,0 +1,244 @@
+package recoveryservices
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/recoveryservicessiterecovery/2022-10-01/replicationpolicies"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/recoveryservices/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+)
+
+type ReplicationPolicyHyperVModel struct {
+	Name                                          string `tfschema:"name"`
+	ResourceGroupName                             string `tfschema:"resource_group_name"`
+	RecoveryVaultName                             string `tfschema:"recovery_vault_name"`
+	RecoveryPointRetentionInHours                 int64  `tfschema:"recovery_point_retention_in_hours"`
+	ApplicationConsistentSnapshotFrequencyInHours int64  `tfschema:"application_consistent_snapshot_frequency_in_hours"`
+	CopyFrequency                                 int64  `tfschema:"replication_interval_in_seconds"`
+}
+
+type ReplicationPolicyHyperVResource struct{}
+
+var _ sdk.ResourceWithUpdate = ReplicationPolicyHyperVResource{}
+
+func (r ReplicationPolicyHyperVResource) Arguments() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringIsNotEmpty,
+		},
+		"resource_group_name": commonschema.ResourceGroupName(),
+
+		"recovery_vault_name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.RecoveryServicesVaultName,
+		},
+		"recovery_point_retention_in_hours": {
+			Type:         pluginsdk.TypeInt,
+			Required:     true,
+			ForceNew:     false,
+			ValidateFunc: validation.IntBetween(0, 24),
+		},
+		"application_consistent_snapshot_frequency_in_hours": {
+			Type:         pluginsdk.TypeInt,
+			Required:     true,
+			ForceNew:     false,
+			ValidateFunc: validation.IntBetween(0, 12),
+		},
+		"replication_interval_in_seconds": {
+			Type:         pluginsdk.TypeInt,
+			Required:     true,
+			ForceNew:     false,
+			ValidateFunc: validation.IntInSlice([]int{30, 300}),
```

Comment:
`ForceNew: false` can be removed
```suggestion
		"recovery_point_retention_in_hours": {
			Type:         pluginsdk.TypeInt,
			Required:     true,
			ValidateFunc: validation.IntBetween(0, 24),
		},
		"application_consistent_snapshot_frequency_in_hours": {
			Type:         pluginsdk.TypeInt,
			Required:     true,
			ValidateFunc: validation.IntBetween(0, 12),
		},
		"replication_interval_in_seconds": {
			Type:         pluginsdk.TypeInt,
			Required:     true,
			ValidateFunc: validation.IntInSlice([]int{30, 300}),
```